MSLaunches.Backend/MSLaunches.Api/Controllers/LunchController.cs

```diff
@@ -76,7 +77,32 @@ public LunchController(ILunchService lunchService)
 
             var result = await _lunchService.CreateAsync(lunchToCreate);
 
-            return CreatedAtAction(nameof(Get), new { userId = result.Id }, new LunchDto(result));
+            return CreatedAtAction(nameof(Get), new { id = result.Id }, new LunchDto(result));
+        }
+
+        [HttpPost]
+        [ValidateModel]
+        [ProducesResponseType(typeof(Lunch), 201)]
+        public async Task<IActionResult> BatchSave([FromBody]List<InputLunchDto> lunches)
+        {
+            // TODO: Fix validation attribute, it's not working as expected.
+            if (lunches == null) return BadRequest();
+
+            var listLunches = new List<Lunch>();
+            foreach (var lunch in lunches)
+            {
+                var lunchToCreate = new Lunch
+                {
+                    Date = lunch.Date,
+                    MealId = lunch.MealId
+
+                };
+                listLunches.Add(lunchToCreate);
+            }
+
+            var result = await _lunchService.CreateLunchesAsync(listLunches);
+
```

Comment:
seems like we always add more lunches, is this correct? if this is the case,  we need to change the signature to AddBatches

Reply:
My misteke including Saving signatures here, was out of scope on this commit. 
I'll add AddBatches on the next PR!